Sources/KituraContracts/Contracts.swift

```diff
@@ -500,7 +500,28 @@ public extension RequestError {
 }
 
 /**
- An identifier for a query parameter object.
+ An object that conforms to QueryParams is identified as being decodable from URLEncoded data.
+ This is used by Codable routes to identify that an object should be initalized by decoding the query parameters
+ using the `QueryDecoder`.
+ ### Usage Example: ###
```

Comment:
- remove trailing ###
- add back-ticks around the code elements
- identify -> indicate
- initalized -> initialized
...actually I'd prefer this sentence:
> This can be applied to a Codable route to define the names and types of the expected query parameters, and provide type-safe access to their values. The `QueryDecoder` is used to decode the URL encoded parameters into an instance of the conforming type.